reference-service/src/main/resources/db/migration/common/V3.80__update_local_office_names.sql

```diff
@@ -0,0 +1,4 @@
+
+UPDATE `LocalOffice` SET `name` = REPLACE(`name`, "Health Education England ", "") WHERE `name` LIKE "Health Education England %";
+UPDATE `Trust` SET `localOffice` = REPLACE(`localOffice`, "Health Education England ", "") WHERE `localOffice` LIKE "Health Education England %";
+UPDATE `Site` SET `localOffice` = REPLACE(`localOffice`, "Health Education England ", "") WHERE `localOffice` LIKE "Health Education England %";
```

Comment:
Please replace `"` double quotes with `'` single quotes.
Double quotes can be used for quoting identifiers.